tsinfer/formats.py

```diff
@@ -2297,9 +2299,9 @@ def __init__(self, path):
         self.path = path
         self.data = zarr.open(path, mode="r")
         genotypes_arr = self.data["call_genotype"]
-        _, self._num_individuals, self.ploidy = genotypes_arr.shape
+        _, self._num_unmasked_individuals, self.ploidy = genotypes_arr.shape
```

Comment:
I interpreted "unmasked" as the number of individuals without the mask set, but instead it's the total number of individuals before masking. Would a better name be e.g. `_total_num_individuals` or `num_individuals_premask`, or something else maybe (also below for `_num_unmasked_samples`